logging/cluster-logging-external-configmap.adoc

```diff
@@ -0,0 +1,26 @@
+:context: cluster-logging-external-configmap
+[id="cluster-logging-external-configmap"]
+= Forwarding logs using a ConfgMap [deprecated]
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+
+You can configure cluster logging to send logs to destinations outside of your {product-title} cluster instead of the default Elasticsearch log store using a ConfigMap:
+
+* *Sending logs using the Fluentd forward protocol*. You can create a Configmap to use the xref:#cluster-logging-collector-fluentd_cluster-logging-external-configmap[Fluentd *forward* protocol] to securely send logs to an external logging aggregator that accepts the Fluent *forward* protocol.
+* *Sending logs using syslog*. You can create a Configmap to use the xref:#cluster-logging-collector-syslog_cluster-logging-external-configmap[syslog protocol] to send logs to an external syslog (RFC 3164) server.
+
+[NOTE]
+====
+Forwarding logs using a ConfigMap will be deprecated when the xref:../logging/cluster-logging-external.adoc#cluster-logging-external[Log Forwarding API] reaches General Availability (GA) status, and will be removed in a future release.
```

Comment:
s/when/once?

Reply:
The IBM Style Guide recommends using _once_ to mean _one time_.